commerce_coordinator/apps/commercetools/clients.py

```diff
@@ -266,6 +269,10 @@ def get_orders(self, customer: CTCustomer, offset=0,
             offset=offset,
             expand=list(expand)
         )
+        logger.info(
+            "[UserOrdersView] [CommercetoolsAPIClient] - Get orders query call finished at %s",
+            datetime.datetime.now()
```

Comment:
@NoyanAziz Can you please put the time started in a variable and subtract it here from the time it concluded so we don't have to manually subtract it to get time in minutes/seconds?

Reply:
Yes let me update with duration for each start and end log